src/structures/cluster.cpp

```diff
@@ -70,6 +70,30 @@ Breakend Cluster::get_average_mate2() const
     return average_breakend;
 }
 
+uint16_t Cluster::get_average_tandem_dup_count() const
+{
+    int64_t sum_counts = 0;
+    int64_t amount_zero_counts = 0;
+    // Iterate through members of the cluster
+    for (size_t i = 0; i < members.size(); ++i)
+    {
+        int16_t current_count = members[i].get_tandem_dup_count();
+        if (current_count == 0) {
+            ++amount_zero_counts;
+        } else {
+            sum_counts += members[i].get_tandem_dup_count();
+        }
+    }
+    // If two thirds of the junctions have a 0 tandem_dup_count, than its probably no tandem duplication.
+    if (amount_zero_counts > std::round(static_cast<double>(members.size()) / 3) )
+    {
+        return 0;
+    } else {
+        int32_t average_count = std::round(static_cast<double>(sum_counts) / members.size());
```

Comment:
The type should match the return type of the function.
```suggestion
        uint16_t average_count = (0.5 + sum_counts) / members.size();
```

or
```suggestion
        return (0.5 + sum_counts) / members.size();
```